diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make origsrc/stardict-2.4.8/src/Makefile.am src/stardict-2.4.8/src/Makefile.am
--- origsrc/stardict-2.4.8/src/Makefile.am	2006-07-02 08:32:16.000000000 -0500
+++ src/stardict-2.4.8/src/Makefile.am	2006-07-13 19:34:27.719000000 -0500
@@ -13,7 +13,7 @@
 
 stardict_DEPENDENCIES = lib/libstardict.a
 stardict_LDFLAGS = 
-stardict_LDADD = $(STARDICT_LIBS) lib/libstardict.a
+stardict_LDADD = lib/libstardict.a $(STARDICT_LIBS)
 
 if !GNOME_SUPPORT
 CONFIG_FILE_MODULE = inifile.cpp inifile.hpp
diff -urN -x CYGWIN-PATCHES -x 'aclocal.m4*' -x ltmain.sh -x 'config.*' -x depcomp -x install-sh -x missing -x mkinstalldirs -x autom4te.cache -x '*compile' -x 'intltool*.in' -x 'xml-i18n-*.in' -x '*.pyc' -x '*.mo' -x '*.gmo' -x ABOUT-NLS -x Makevars.template -x COPYING -x INSTALL -x '*.orig' -x '*.rej' -x '*~' -x '*.temp' -x texinfo.tex -x ylwrap -x 'Makefile.in*' -x configure -x gnome-doc-utils.make -x intltool.m4 -x omf.make -x xmldocs.make origsrc/stardict-2.4.8/tests/Makefile.am src/stardict-2.4.8/tests/Makefile.am
--- origsrc/stardict-2.4.8/tests/Makefile.am	2006-06-08 03:40:41.000000000 -0500
+++ src/stardict-2.4.8/tests/Makefile.am	2006-07-13 19:41:34.031500000 -0500
@@ -23,7 +23,7 @@
 t_lookupdata_SOURCES = t_lookupdata.cpp
 t_lookupdata_DEPENDENCIES = $(top_builddir)/src/lib/libstardict.a
 
-LDADD = @STARDICT_LIBS@ $(top_builddir)/src/lib/libstardict.a
-INCLUDES = @STARDICT_CFLAGS@  -I$(top_builddir) -I$(top_builddir)/src -I$(top_builddir)/src/lib
+LDADD = $(top_builddir)/src/lib/libstardict.a $(STARDICT_LIBS)
+INCLUDES = $(STARDICT_CFLAGS)  -I$(top_builddir) -I$(top_srcdir)/src -I$(top_srcdir)/src/lib
 
 TESTS = t_config_file t_dict t_query t_lookupdata
